JMdictFiles/sameKanjiCounter.js

```diff
@@ -0,0 +1,51 @@
+const { JM } = require('./dic');
```

Comment:
again looks like we are missing some files in the repo? Note that empty folders will not be picked up by git.